spec/components/course/model_component_host_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Course::ControllerComponentHost do
+  class ADummyComponent
+    include Course::ModelComponentHost::Component
+
+    cattr_accessor :called
+    @@called = false
+
+    def self.course_initialize(course)
```

Comment:
Unused method argument - `course`. If it's necessary, use `_` or `_course` as an argument name to indicate that it won't be used. You can also write as `course_initialize(*)` if you want the method to accept any arguments but don't care about them.